kolena/_utils/batched_load.py

```diff
@@ -112,10 +112,10 @@ def concat(dfs: Iterable[pd.DataFrame], df_class: Type[DFType]) -> DFType:
         return df_class(df)
```

Comment:
There's a mypy error on this line, I think it can be resolved by `return df`?

Reply:
Resolved by having `LoadableDataFrame` extend `pd.DataFrame` — which it probably should have done all along.